src/Tribe/Views/V2/View.php

```diff
@@ -1081,8 +1081,10 @@ protected function setup_template_vars() {
 			'prev_label'           => $this->get_link_label( $this->prev_url( false ) ),
 			'next_label'           => $this->get_link_label( $this->next_url( false ) ),
 			'date_formats'         => (object) [
-				'compact'        => Dates::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
-				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
+				'compact'              => Dates::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
+				'month_and_year'       => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
+				'time_range_separator' => tribe_get_date_option( 'timeRangeSeparator', ' - ' ),
+				'date_time_separator'  => tribe_get_date_option( 'dateTimeSeparator', ' @ ' ),
```

Comment:
question because I don't know, do the separators always have the spaces around them, or are they just the `-` or `@` characters?

Reply:
I'm also not sure if this is the right place to put this, as this technically isn't a date format. @bordoni thoughts?

Reply:
Default values are with spaces: https://github.com/moderntribe/the-events-calendar/search?q=dateTimeSeparator&unscoped_q=dateTimeSeparator

Reply:
I feel like either way is fine here. In the placement of the actual formatting. What do you think @lucatume ??
Any particular opinions?

Reply:
I think it's fine to have it here, while a bit stretched, the `date_formats` array could either contain only date formats, or loosely contain anything required to format dates.
I this case I think this is a good solution.